renku_notebooks/config.py

```diff
@@ -56,6 +56,12 @@
 """The default image to use for an interactive session if the image tied to the
 current commit cannot be found."""
 
+NOTEBOOKS_USE_PERSISTENT_VOLUMES = os.environ.get(
```

Comment:
I might have missed this, but it seems to me that this config is not used in the server_options endpoint, so could it be that if this is set to false we're advertising the disk option to the UI anyway?

Reply:
good point, should filter it out there. otoh we may want to impose limits on ephemeral volumes as well...